index.src.html

```diff
@@ -613,6 +613,11 @@
         failing to release them (or waiting for garbage collection) may cause
         decoding to stall.
 
+    NOTE: {{VideoDecoder}} requires that frames are output in the order they
+        expect to be presented, commonly known as presentation order. Some
+        {{VideoDecoder/[[codec implementation]]}}s may require the User Agent
+        to reorder outputs into presentation order.
```

Comment:
This can be made normative, defining what presentation order means, and linked in the steps below.